src/com/tony/server/Main.java

```diff
@@ -38,12 +38,15 @@ public static void main(String[] args) {
                 uriList.add("/" + listOfFiles[i].getName());
             }
         }
+        uriList.remove("/.DS_Store");
+        uriList.remove("/form");
+        uriList.remove("/method_options");
         return uriList;
```

Comment:
I thought this code seemed a little shady as well. 

But I added it because when you GET "/", the .DS_Store, /form, and /method_options were showing up as links as well since they (form and method_options) were created by cob_spec and never deleted by cob_spec.

I guess they should indeed show up in GET "/" since they are files that exist in the public folder. I can get rid of these 3 lines. What do you think?